packages/@apphosting/adapter-angular/src/simple-server/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "scripts": {
-      "build": "rollup -c --bundleConfigAsCjs"
+      "generate-server": "rollup -c --bundleConfigAsCjs"
```

Comment:
what is this change for

Reply:
Previously the build was breaking because this was build (it was being picked up by the lerna build) but I reverted it and it seems to be working fine so I switched it back.